loadtests/src/test/main/scala/svp/SvpSimulationConstantUsersPerSec.scala

```diff
@@ -0,0 +1,17 @@
+package svp
+
+import io.gatling.core.Predef._
+import scala.concurrent.duration._
+
+class SvpSimulationConstantUsersPerSec extends Simulation {
+  setUp(ShieldedVulnerablePeople.scnEndToEndJourney.inject(
+    constantUsersPerSec(Config.injectUsersPerSecond) during (Config.injectDurationSeconds seconds)))
+    .protocols(Config.httpProtocol)
+    .maxDuration(Config.duration * 60)
```

Comment:
this duration can be just in minutes now without the * 60

Reply:
done